groups/xml.yml

```diff
@@ -0,0 +1,3 @@
+# Group comprised of XML features such as parsing/serializing, querying and more.
+# It does not include XMLHTTPRequest, since it's not specific to XML.
```

Comment:
Good shout on XMLHTTPRequest here.

```suggestion
# Group comprises XML features such as parsing/serializing, querying and more.
# It does not include XMLHTTPRequest, since it's not specific to XML.
```